crates/bevy_ecs/src/archetype.rs

```diff
@@ -35,6 +35,7 @@ impl ArchetypeId {
     }
 }
 
+#[derive(Copy, Clone)]
 pub(crate) enum ComponentStatus {
```

Comment:
Basic docs here to help explain how this new trait is being used?

Reply:
I'm slightly confused by this comment. This is just making an existing internal type Copy/Clone.

Reply:
I'm guessing you meant this comment to be on BundleComponentStatus. I've added clarifying documentation to the behavior and role of this trait.